java/serving/src/main/java/feast/serving/config/ApplicationProperties.java

```diff
@@ -246,10 +249,31 @@ public void setType(String type) {
      * @return Returns the store specific configuration
      */
     public RedisClusterStoreConfig getRedisClusterConfig() {
+      String read_from = ReadFrom.UPSTREAM.toString();
+      if (!this.config.containsKey("read_from") || this.config.get("read_from") == null) {
+        log.info("'read_from' not defined in Redis cluster config, so setting to UPSTREAM");
```

Comment:
Sorry this is a bit nitpicky, but isnt it safer to have the variable be set within the `if` statement? It's easier to read, and less likely to be refactored erroneously. Same for the below if statement.

Reply:
Done